check-swlib.sh

```diff
@@ -127,8 +127,11 @@ done
 }
 
 # Oracle Database free edition parameter overrides
-if [[ "${ORA_EDITION}" = "FREE" && ! "${ORA_VERSION}" =~ ^23\. ]]; then
+if [ "${ORA_EDITION}" = "FREE" ]; then
+  ORA_DISK_MGMT=FS
```

Comment:
If we're fetching free edition via URI, we wouldn't use `--ora-swlib-bucket` at all right?

Reply:
If there a relatively straightforward to make it optional in the appropriate cases?